### PR TITLE
Update: Revert Mil2525DMessages.xml changes with Ordered Anchor Points support

### PR DESCRIPTION
**Pull request by yo1995**
## Description

This reverts commit 1db25cf2a6b124efb1d001531f0c46ecb4e9d2dc (https://github.com/Esri/arcgis-runtime-samples-ios/pull/739), which reverts the changes in the `Mil2525DMessages.xml` XML file.

Other changes included

- Change basemap style to make it visually clearer
- Change from using local style file to portal item
- Included the "ORDERED ANCHOR POINTS" request from @pMaske and @kerryrobinson 

## Linked Issue(s)

- `common-samples/issues/3003`
- `common-samples/issues/3003#issuecomment-3213191`

## How To Test

- Verify `Graphics Overlay (Dictionary Renderer) 3D sample` is still working, and the symbols and shapes are all looking correct.

## To Discuss

- Please leave suggestions for the `How it works` section of the README, so that we can fit the "ordered anchor points" attribute into it. I wasn't sure how to word it... 😅

### CODE REVIEW

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist
Comment (yo1995) [inline arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist:1036]: Because we load from portal instead of local file, I removed the dependency for the sample. However, the `mil2525d` file is also used by `FeatureLayerDictionaryRendererViewController`, so I don't remove it from the project. Maybe one day it can also be updated to using the portal item.

File: arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/GraphicsOverlayDictionaryRenderer3DViewController.swift
Comment (yo1995) [inline arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/GraphicsOverlayDictionaryRenderer3DViewController.swift:26]: I used the topo base style here because it makes the symbology stand out. Also to show off the `AGSScene` vector basemap support in U13.

**Comment by yo1995** [comment]
@pMaske , @kerryrobinson It would be appreciated if you can take a brief look at the changes in this PR, to double check we are getting it correct. 👀

**Review by vquach2404 — APPROVED** [review]
Looks good! Don't forget to update the screenshot since the basemap has changed

**Comment by yo1995** [comment]
> Looks good! Don't forget to update the screenshot since the basemap has changed

Thanks for the reminder! Replaced. After Phil's review, I'll hold this PR for either Kerry or Preeti's comment, before merging into `v.next`.

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist
Comment (philium) [inline arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist:1035]: Now that the sample is using a portal item, this description ("using a local…style file") is no longer correct.

File: arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/GraphicsOverlayDictionaryRenderer3DViewController.swift
Comment (philium) [inline arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/GraphicsOverlayDictionaryRenderer3DViewController.swift:32]: Does this need to be a property? If so, I'm curious why. If not, please move it into `makeGraphicsOverlay()`.

File: arcgis-ios-sdk-samples/Shared resources/Military messages/Mil2525DMessages.xml
Comment (philium) [inline arcgis-ios-sdk-samples/Shared resources/Military messages/Mil2525DMessages.xml:9]: It seems a bit odd that this file is being updated. All the other changes are specific to the Graphics Overlay (Dictionary Renderer) sample, but that sample no longer uses this file. Including these changes in this PR means that two samples have to be tested instead of one.

Comment (yo1995) [reply arcgis-ios-sdk-samples/Shared resources/Military messages/Mil2525DMessages.xml:9]: The sample still uses this file, see [line 63](https://github.com/Esri/arcgis-runtime-samples-ios/pull/1128/files#diff-f137eb0a526906d151086880c0c080aa8282bfcd21380095b7478700f097e158R63). Basically to render the symbols, we need... 

- the stylx file, which contains the symbols
- the xml file, which contains the geometry, such as control points for polylines, single points to place markers, etc.

These reverted changes in the xml file are requested by symbology team, in order to support the `Ordered Anchor Points` attribute in the style.

Comment (philium) [reply arcgis-ios-sdk-samples/Shared resources/Military messages/Mil2525DMessages.xml:9]: Oh! This isn't the style file. Right. Never mind!

File: arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/GraphicsOverlayDictionaryRenderer3DViewController.swift
Comment (yo1995) [reply arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/GraphicsOverlayDictionaryRenderer3DViewController.swift:32]: Thanks, no need to be a property.

File: arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist
Comment (yo1995) [reply arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist]: I made some slight adjustments to the README and description.

File: arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/README.md
Comment (philium) [inline arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/README.md:17]: A comma isn't needed here.
```suggestion
1. Create a new `AGSDictionarySymbolStyle` object with the "mil2525d" specification type and set the style's draw rule configuration to "ORDERED ANCHOR POINTS".
```

**Review by philium — COMMENTED** [review]
Looks good! Just one more suggestion.

**Review by philium — APPROVED** [review]

**Comment by pMaske** [comment]
@yo1995 looks good to me

**Review by saratchandrakarumuri — DISMISSED** [review]
Update looks good

**Comment by yo1995** [comment]
Thanks all! I had to make a last minute adjustment to the README's description suggested by Kerry.

**Review by saratchandrakarumuri — APPROVED** [review]

**Review by vquach2404 — APPROVED** [review]

**Pull request merged**
